modules/home-environment.nix

```
@@ -303,6 +303,15 @@ in
         Extra commands to run in the Home Manager profile builder.
       '';
     };
+
+    home.targetOperatingSystem = mkOption {
+      type = types.enum [ "nixos" "linux" ];
+      default = "nixos";
+      example = "linux";
+      description = ''
+        The operating system on which this configuration will be deployed.
```

To stay consistent with the wording we have in the default `home.nix`.

```suggestion
        The host operating system that Home Manager will be installed on. This
        enables OS-specific workarounds and tweaks to increase the general
        cohesiveness of the system.
```

---

Fixed